recipes/_omnibus_toolchain.rb

```diff
@@ -39,6 +39,8 @@
   end
 else
   chef_ingredient node['omnibus']['toolchain_name'] do
+    platform node['platform_family']
```

Comment:
This won't work correctly on EL nodes. You will need to map `rhel` to `el` and a platform version like `6.5` to `6`. Running a similar query via the mixlib-install CLI shows the issue:

```
➜  mixlib-install download chef --platform rhel --platform-version 6.5 --url
Querying for artifact with options:
{
  "channel": "stable",
  "product_name": "chef",
  "product_version": "latest",
  "platform_version_compatibility_mode": true,
  "platform": "rhel",
  "platform_version": "6.5",
  "architecture": "x86_64"
}
No results found.

➜  mixlib-install download chef --platform el --platform-version 6 --url
Querying for artifact with options:
{
  "channel": "stable",
  "product_name": "chef",
  "product_version": "latest",
  "platform_version_compatibility_mode": true,
  "platform": "el",
  "platform_version": "6",
  "architecture": "x86_64"
}
https://packages.chef.io/files/stable/chef/12.20.3/el/6/chef-12.20.3-1.el6.x86_64.rpm
```

@wrightp We need to add some of this Ohai platform/platform version values to expected mixlib-install platform/platform version into `chef_ingredient`.